### PR TITLE
fix windows error on `std::os::unix::ffi` and `as_os_str().as_bytes()`

### PR DESCRIPTION
**Pull request by diniremix**
added the `cfg target_os` attribute,for windows support

[streamable video: https://streamable.com/n7gwjp](https://streamable.com/n7gwjp)

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -80,10 +85,16 @@ fn main() -> anyhow::Result<()> {
 
         let path = file.path();
 
+        #[cfg(target_os = "linux")]
```
Comment (fprasx) [inline src/main.rs:88]: Could you change this to `#[cfg(not(target_of = "windows"))]` so this also works on mac?

**Review by fprasx — COMMENTED** [review]
Left a comment, otherwise all good.

**Comment by diniremix** [comment]
done!, thanks for checking 👍

**Comment by fprasx** [comment]
Awesome, thank you for the PR!

**Pull request merged**